loki-client/src/main/java/com/github/loki4j/client/pipeline/PipelineConfig.java

```diff
@@ -187,6 +196,17 @@ public static class Builder {
         private long sendQueueMaxBytes = batchMaxBytes * 10;
         private int maxRetries = 2;
         private long retryTimeoutMs = 60 * 1000;
+        private Supplier<Long> jitter = new Jitter(1000);
```

Comment:
Please use a simple property here, e.g. `maxJitterMs`. An instance of Jitter should be created in Pipeline

Reply:
Moved logic to Jitter.